packages/shared/src/utils/string/string_util.js

```diff
@@ -1,6 +1,14 @@
 export const toTitleCase = str =>
     (str || '').replace(/\w[^\s/\\]*/g, txt => txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase());
 
+export const toSentenceCase = str => {
+    const string = str || '';
+    const replace_filter = string.replace(/-|_/g, ' ');
+    const sentenced_case = replace_filter[0].toUpperCase() + replace_filter.slice(1).toLowerCase();
+
+    return sentenced_case;
+};
+
```

Comment:
We should not be using such a utility, it is practically impossible to catch all cases, it will have issues like `toTitleCase` does. API needs to present a `display` field for it. In the meantime, we can perhaps change this into a `formatFields` function, and only use it for the use-case of this PR (or the API).